tasks/src/audio-to-audio/about.md

```diff
@@ -32,13 +32,13 @@ data = query("sample1.flac")
 
 ### Audio Source Separation
 
-Audio Source Separation allows you to isolate the different sounds from the individual sources. For example, if you have an audio file with multiple people speaking, you can get an audio file for each of them. You can then use an Automatic Speech Recognition system to extract the text from each of these sources as an initial step for your system!
+Audio Source Separation allows you to isolate different sounds from individual sources. For example, if you have an audio file with multiple people speaking, you can get an audio file for each of them. You can then use an Automatic Speech Recognition system to extract the text from each of these sources as an initial step for your system!
 
-Audio to audio can also be used to remove noise from audio files: you get one audio for the person that is speaking and another audio for the noise. This can also be helpful when you have an audio of multiple people with some noise: you can get an audio for each person and then an audio for the noise.
+Audio-to-Audio can also be used to remove noise from audio files: you get one audio for the person speaking and another audio for the noise. This can also be useful when you have multi-person audio with some noise: yyou can get one audio for each person and then one audio for the noise.
```

Comment:
```suggestion
Audio-to-Audio can also be used to remove noise from audio files: you get one audio for the person speaking and another audio for the noise. This can also be useful when you have audio from multiple people with some noise: you can get one audio for each person and then one audio for the noise.
```